sdccc/src/test/java/com/draeger/medical/sdccc/manipulation/precondition/PreconditionRegistryTest.java

```diff
@@ -46,6 +48,9 @@ public void testPreconditionInteractionRegistrationAndPrompt() throws Exception
         final var mockInjector = mock(Injector.class);
         final var registry = new PreconditionRegistry(mockInjector);
 
+        final var messageStorageMock = mock(MessageStorage.class);
```

Comment:
Just add the messageStorageMock to the setUp method, instead of adding it to every unit test.

Reply:
i also added the the injector and registry to setup